src/main/java/taxi/dao/impl/CarDaoJdbcImpl.java

```diff
@@ -0,0 +1,195 @@
+package taxi.dao.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import taxi.dao.CarDao;
+import taxi.lib.Dao;
+import taxi.model.Car;
+import taxi.model.Driver;
+import taxi.model.Manufacturer;
+import taxi.util.ConnectionUtil;
+
+@Dao
+public class CarDaoJdbcImpl implements CarDao {
+    @Override
+    public Car create(Car car) {
+        String sqlQuery = "INSERT INTO cars (cars_model, manufacturers_id)"
+                + "VALUES (?,?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement preparedStatement = connection
+                        .prepareStatement(sqlQuery, Statement.RETURN_GENERATED_KEYS)) {
+            preparedStatement.setString(1, car.getModel());
+            preparedStatement.setLong(2, car.getManufacturer().getId());
+            preparedStatement.executeUpdate();
+            ResultSet resultset = preparedStatement.getGeneratedKeys();
+            while (resultset.next()) {
+                car.setId(resultset.getObject(1, Long.class));
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException("Can't insert car " + car, e);
+        }
+        return car;
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        String sqlQuery = "SELECT * FROM cars INNER JOIN manufacturer "
+                + "ON manufacturers_id = manufacturer_id "
+                + "WHERE cars_deleted = false AND cars_id =?;";
```

Comment:
let's use aliases and specify the table(alias) for each field

Reply:
done